RadixWallet/Features/SettingsFeature/DisplayMnemonics/Coordinator/DisplayMnemonics+View.swift

```diff
@@ -42,8 +42,8 @@ extension DisplayMnemonics {
 				}
 				.background(.app.gray5)
 				.radixToolbar(title: L10n.SeedPhrases.title)
-				.onFirstTask { @MainActor in
-					await viewStore.send(.onFirstTask).finish()
+				.onAppear {
```

Comment:
Reason for changing from task to appear?

Reply:
If you got to this view with a problem, opened `DisplayMnemonic.View` and took care of it; the problem would still show up when getting back to this view. 
I also expected it to update automatically with the async sequence, but I believe it doesn't since values aren't stored anywhere (and we need to trigger the check again).

Reply:
> and we need to trigger the check again
Sure, then you could use just `task` not `onFirstTask`.